qemu/tests/qsd_qmp_cmd_check.py

```diff
@@ -0,0 +1,92 @@
+"""QSD QMP commands test"""
+import json
+
+from provider.qsd import QsdDaemonDev
+from virttest import error_context
+from virttest.qemu_monitor import QMPCmdError
+
+
+@error_context.context_aware
```

Comment:
"error_context.context()" statement was not referenced in codes, so I think we can remove line 5 and line 9. If I'm wrong, please correct me. Thanks.

Reply:
updated